flag_test.go

```diff
@@ -2050,3 +2050,34 @@ func TestTimestampFlagApply_WithDestination(t *testing.T) {
 	expect(t, err, nil)
 	expect(t, *fl.Destination.timestamp, expectedResult)
 }
+
+// Test issue #1254
+// StringSlice() with UseShortOptionHandling causes duplicated entries, depending on the ordering of the flags
+func TestSliceShortOptionHandle(t *testing.T) {
+	_ = (&App{
+		Commands: []*Command{
+			{
+				Name:                   "foobar",
+				UseShortOptionHandling: true,
+				Action: func(ctx *Context) error {
```

Comment:
One of the things that makes me nervous about only having assertions in a callback function is that the test passes if the callback function is never called. What I typically do in this kind of situation is add a boolean check:

```go
wasCalled := false

// ...

Action: func(ctx * Context) error {
  wasCalled = true
  // ...
},

// ...

// at the very end of the function
if !wasCalled {
  t.Error("missing call to action function")
}
```

Reply:
Done.